content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Camel Quarkus 1.0.0 Released"
+date: 2020-08-10
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: The first stable release of Camel Quarkus
+summary: Walk through the highlights of the first stable release: Developer joy, Camel component coverage, Bootstrap, CDI, native mode and more!
```

Comment:
@zregvart I hope this would work as a summary?

Reply:
Looks [good](https://deploy-preview-454--camel.netlify.app/blog/).